src/config.rs

```diff
@@ -51,10 +52,11 @@ mod tests {
 
     #[test]
     fn test_config_parsing() {
-        let config_str = std::fs::read_to_string("./config/config.toml").unwrap();
+        let config_str =
+            std::fs::read_to_string("./config/config.toml").unwrap();
         let config: Config = toml::from_str(&config_str).unwrap();
 
         assert_eq!(config.debug, false);
```

Comment:
You've [turned the debug on](https://github.com/AttorneyOnline-Unofficial/rusttorney/pull/4/files#diff-8f70968e4ceb280a79c233b1fc620aebR1) so this will fail

Reply:
Maybe it's better to just parse a string literal instead of a file?